src/libs/API.js

```diff
@@ -149,6 +150,8 @@ Network.registerRequestSkippedHandler((parameters) => {
 });
 
 Network.registerResponseHandler((queuedRequest, response) => {
+    NetworkResponseManager.publish(queuedRequest.command, {request: queuedRequest, response});
```

Comment:
<img width="274" alt="image" src="https://user-images.githubusercontent.com/1228807/156032090-cf27e20d-0fb4-4975-bab5-a384701c66ce.png">

More of a naming conversation, but this seems pretty confusing and the separation is not really intuitive.

I don't have any other suggestions for this now, but I wanted to pull it out as something to think about.

Reply:
Agree it could be incorporated into network. The separation just helped me compartmentalize the problem and just show off the "new code" we could be adding.